src/botcmd.c

```diff
@@ -697,7 +696,7 @@ static void bot_nlinked(int idx, char *par)
     dprintf(idx, "error %s\n", BOT_YOUREALEAF);
   }
   if (s[0]) {
-    chatout("*** %s\n", s);
+    putlog(LOG_BOTS, "*", "%s.", s);
```

Comment:
For standardization purposes, can we leave the proceeding '***'s of the message in place here (and elsewhere)? It would be good to retain as much of the previous appearance/functionality as possible.